packages/fuels/src/cli.ts

```diff
@@ -66,6 +67,11 @@ export const configureCli = () => {
     .addOption(pathOption)
     .action(withConfig(command, Commands.dev, dev));
 
+  (command = program.command(Commands.node))
+    .description('Start a Fuel node')
```

Comment:
```suggestion
    .description('Start a Fuel-Core node')
```

Reply:
I suppose it would be `fuel-core` then.

Reply:
The description for the `dev` command says `Fuel`, so I went with that here as well.

Reply:
I think `Fuel` node would be more intelligible by the users of this command @danielbate , whilst we do make that distinction within the naming of the packages, there is only one implication of a _Fuel node_ to the best of my knowledge.